myproject/myproject/urls.py

```diff
@@ -1,8 +1,14 @@
 from django.contrib import admin
 from django.urls import include, path
+from django.conf import settings
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('blog/', include("blog.urls")),
     path('users/', include("users.urls")),
 ]
+if settings.DEBUG_TOOLBAR == 1:
```

Comment:
You don't need to explicitly write the `== 1`. Just write `if settings.DEBUG_TOOLBAR` then run the inside code.